src/rom800D674.c

```diff
@@ -0,0 +1,46 @@
+#include "global.h"
+#include "main.h"
+
+void sub_800D674(void)
+{
+    u32 i;
+    u8 *roomptr = gMain.roomData[gMain.currentRoomId];
+    roomptr += 4;
+    for(i = 0; i < 4; i++) {
+        u8 *src;
+        void *destination = (void *)VRAM+0x13400;
+        destination += i*0x800;
+        if(*roomptr != 0xFF) {
+            src = gUnknown_081FD96C+*roomptr*0x800;
+            DmaCopy16(3, src, destination, 0x800);
+        }
+        roomptr++;
+    }
+}
+
+void sub_800D6C8(void)
+{
+    u32 i;
+    struct TalkData *talkdata;
+    u8 *fillerdata;
+    for(talkdata = gTalkData; talkdata->unk0 != 0xFF; talkdata++) {
+        if(gMain.currentRoomId == talkdata->unk0) {
+          if(gAnimation[1].unkC.unk2[0] == talkdata->unk1) {
+              if(talkdata->unk3 == 1) break;
+          }
+        }
+    }
+    fillerdata = talkdata->filler4;
```

Comment:
Data in structs marked as filler is just parts of the struct that are not known to be used by anything and this function is using them therefore it has to be turned into its own field in the struct